### PR TITLE
Benchmarks for azk cli

### PR DESCRIPTION
**Pull request by saitodisse**
There are two new ways to mesure `azk`'s performance:

## Mesure time and send to keen-io

This script must be run every time we commit on `master` branch. It runs the most used azk commands using `azk demo` project. At end sends all infos to keen.io.

```sh
$ ./src/libexec/profiling/simple.sh
```

## Profiling CPU with chrome-cpu-profiler

```sh
# from azk's root
$ ./src/libexec/profiling/cpu-profiling.sh
```

You can open the generated `.cpuprofile` files with Googles Chrome Profiler. Chrome -> Developer Tools -> Profiles -> Load Button

## See timeline with njstrace

```sh
# from azk's root
$ ./src/libexec/profiling/njs-tracer.sh
```

This generates a long json file, that will be analysed with some computation.

### CODE REVIEW

**Comment by nuxlli** [comment]
Haven't ran the scripts, but two questions caught my attention:

- Scripts of this style (such as packaging) should be placed in `.src/libexec` and not in the folder `./shared`, since the `./shared` folder is packed up when distributing the `azk` and `./src` folder not;

- Benchmarks added are code benchmark, we still need command's benchmark scripts:

```
azk version
azk info
azk status
azk start
azk agent start
```

**Comment by saitodisse** [comment]
@nuxlli, i will move these files to `.src/libexec`.

`shared/scripts/cpu-profiling.sh` generates the command benchmarks in a separate file. Unless the `azk start`, all commands are been benchmarked.

**Comment by heitortsergent** [comment]
@nuxlli @saitodisse I made some changes but I'm not too familiar with bash. Can you guys point me in the right direction on how to make this better? :dancer:

File: src/libexec/profiling/njs-tracer.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+# cleanup
+azk agent stop
+
+# RUN all commands
+run 'azk version'
+run 'azk agent start'
+run 'azk info'
+run 'azk status'
+run 'azk agent stop'
+# run 'azk start' #FIXME: do a folder to test azk start
```
Comment (heitortsergent) [inline src/libexec/profiling/njs-tracer.sh]: @saitodisse does this still needs fixing?

File: src/libexec/profiling/simple.sh [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+run 'adocker version'
+run 'adocker info'
+run 'azk version'
+run 'azk agent start'
+run 'azk info'
+run 'azk init'
+run 'azk start'
+run 'azk stop'
+run 'azk status'
+run 'azk agent stop'
```
Comment (heitortsergent) [inline src/libexec/profiling/simple.sh]: Lines 55 and 56 need to be moved to after "azk agent start" to work on Mac

File: src/libexec/profiling/njs-tracer.sh (hunk 1 above)
Comment (saitodisse) [inline src/libexec/profiling/njs-tracer.sh]: no, not anymore. I forgot to remove this FIXME

File: src/libexec/profiling/simple.sh (hunk 2 above)
Comment (saitodisse) [inline src/libexec/profiling/simple.sh]: Ok, I will do that

**Pull request merged**